docs/install-w-machine.md

```diff
@@ -1,231 +1,173 @@
 <!--[metadata]>
 +++
-title = "Install and use Swarm"
-description = "Swarm release notes"
-keywords = ["docker, swarm, clustering, discovery, release,  notes"]
+title = "Getting started with Docker Swarm"
+description = "Introduction to Swarm commands and concepts"
+keywords = ["docker, swarm, clustering, discovery, examples"]
 [menu.main]
 parent="workw_swarm"
 +++
 <![end-metadata]-->
 
-# Install and Create a Docker Swarm
+# Swarm get started (novice)
 
-You use Docker Swarm to host and schedule a cluster of Docker containers. This section introduces you to Docker Swarm by teaching you how to create a swarm
-on your local machine using Docker Machine and VirtualBox.
+## Introduction
 
-## Prerequisites
+This getting started example shows you how to create a Docker Swarm, the
```

Comment:
This is a bit confusing, they aren't creating a Docker Swarm they are creating a cluster of hosts running Docker Engine. Docker Swarm the product allows the cluster to be addressed through the Docker Engine Remote API.